lib/srv/term.go

```diff
@@ -168,10 +168,8 @@ func (t *terminal) AddParty(delta int) {
 }
 
 // Run will run the terminal.
-func (t *terminal) Run(ctx context.Context) error {
+func (t *terminal) Run(_ context.Context) error {
 	var err error
-	defer t.closeTTY()
```

Comment:
We're closing the `TTY` in two different places anyway. Closing it here causes the return code to be "skipped".